app/src/main/java/com/macbitsgoa/events/aboutfest/AboutFestActivity.java

```diff
@@ -0,0 +1,54 @@
+package com.macbitsgoa.events.aboutfest;
+
+import android.os.Bundle;
+
+import com.facebook.drawee.backends.pipeline.Fresco;
+import com.google.android.material.tabs.TabLayout;
+import com.macbitsgoa.events.R;
+
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentActivity;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentPagerAdapter;
+import androidx.viewpager.widget.ViewPager;
+
+public class AboutFestActivity extends FragmentActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_about_fest);
+
+        //Initialize Fresco
+        Fresco.initialize(this);
```

Comment:
put this initialization in application class 
> app/src/main/java/com/macbitsgoa/events/Events.java
```
public class Events extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        Fresco.initialize(this);
    }
}
```
Also add the
```xml
      android:name=".Events"
```
attribute in `AndroidManifest.xml` file.